server/container/jetty/src/main/java/org/apache/james/http/jetty/Configuration.java

```diff
@@ -135,6 +135,13 @@ public Builder port(int port) {
             return this;
         }
 
+        public Builder port(Optional<Integer> port) {
```

Comment:
what is the point accepting an optional ?
It duplicates other `port()` method, too